crates/solver/src/driver.rs

```diff
@@ -475,27 +485,26 @@ impl Driver {
 
         let mut solver_settlements = Vec::new();
 
+        let next_solver_competition = auction.next_solver_competition;
         let auction = Auction {
-            id: auction_id,
+            id: auction.next_solver_competition,
+            run: run_id,
             orders: orders.clone(),
             liquidity,
             gas_price: gas_price.effective_gas_price(),
             deadline: Instant::now() + self.solver_time_limit,
             external_prices: external_prices.clone(),
         };
 
-        tracing::debug!("solving auction id {}", auction.id);
+        tracing::debug!(deadline =? auction.deadline, "solving auction");
```

Comment:
no point in logging the ID (since it is redundant information included int he `info_span!`) and the other fields of the `Auction` have already been logged *except* for the deadline. So I repurposed this log line to include the deadline.